plugins/napari_ADS/_widget.py

```diff
@@ -183,6 +183,7 @@ def __init__(self, napari_viewer):
 
         load_mask_button = QPushButton("Load mask")
         load_mask_button.clicked.connect(self._on_load_mask_button_click)
+        self.load_mask_button = load_mask_button

```

Comment:
This attribute doesn't seem to be used anywhere else in the plugin. Can we remove this line?

Reply:
Good catch - fine with me! Don't recall why I added that